tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
```suggestion
```
This line should be removed.  @tokers @nic-6443

Reply:
Since this is a bug, I hope to merge it into v1.4 as soon as possible, and I will continue to complete this PR. cc @tokers @nic-6443 Thanks for your contribution.